ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -0,0 +1,126 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using EFAuditer;
+using ntbs_service.DataAccess;
+using ntbs_service.Models;
+using ntbs_service.Models.Enums;
+using Serilog;
+
+namespace ntbs_service.DataMigration
+{
+
+    public interface INotificationImportService
+    {
+        Task<IList<Notification>> ImportNotificationsAsync(string notificationId);
+    }
+
+    public class NotificationImportService : INotificationImportService
+    {
+        private readonly INotificationSearcher notificationSearcher;

```

Comment:
Can we please make private fields named in the convention `_notificationSearcher` - this also means you don't need to bother with `this.` in the constructor.